templates/default.mustache

```diff
@@ -141,6 +141,13 @@ elseif has("terminal")
         \ "#{{base07-hex}}",
         \ ]
 endif
+if exists("base16background") && base16background == "none"
```

Comment:
The name of the option suggests that any color can be set:

`let base16background="#eaebeb`

Could you come up with a name that doesn't imply this? For example:

```suggestion
if base16backgroundnone
```

Where `base16backgroundnone` defaults to `0` (this would also let us skip the `exists` check).

Reply:
I would go with:

`base16_transparent_background` this makes clear what the variable does, `none` is more a implementation detail. 

and changing `base16colorspace` to `base16_colorspace` for consistency

Reply:
I'd argue for adding `base16_colorspace` but not removing `base16colorspace` to avoid breaking people's vims.